spec/graphql/queries/categories/get_categories_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'get all categories', type: :request do
+  describe 'find list of categories' do
```

Comment:
It's great to see your testing syntax. It looks so similar to what we see in the front end!